specification/deviceregistry/DeviceRegistry.Management/tspconfig.yaml

```diff
@@ -24,7 +24,7 @@ options:
     package-dir: "azure-resourcemanager-deviceregistry"
     flavor: "azure"
     service-name: "Device Registry"
-    api-version: "2024-09-01-preview"
+    api-version: "2024-11-01"
```

Comment:
You can delete this line, if it is already the latest in tsp.

Reply:
![image](https://github.com/user-attachments/assets/d838c416-5b8b-4e14-bcb8-2c17968cbe5c)
Is this the version record of this part?

Reply:
yes, the version in main